File: Kshitiz/Boy_Or_Girl.py
s=sorted(list(input()))
x,a,n=0,0,len(s)
while x<n:
	sn=s.count(s[x])
	if sn>1:
		x+=sn
	else:
		x+=1
	a+=1
if a%2==0:
	print('CHAT WITH HER!')
else:
	print('IGNORE HIM!')

File: Kunal/Boy_or_Girl.cpp
#include<bits/stdc++.h>
using namespace std;
int main(){
    char s[100];
    int f=0,a=0;
    cin >> s;
    for (int i = 0; s[i]!='\0'; i++)
    {
        for (int j = i-1; j >=0; j--)
        {
            if(s[i]==s[j])
            {
                f=1;
                break;
            }
        }
        if(f==0)
        {
            a++;
        }
        f=0;  
    }
    if(a%2==0)
    {
        cout << "CHAT WITH HER!" << endl;
    }
    else cout << "IGNORE HIM!" <<endl;
    return 0;
}